src/Sonarr.Http/ApiInfoController.cs

```diff
@@ -0,0 +1,34 @@
+using Microsoft.AspNetCore.Mvc;
+using NzbDrone.Core.Configuration;
+
+namespace NzbDrone.Http
+{
+    public class ApiInfoController : Controller
+    {
+        private readonly IConfigFileProvider _configFileProvider;
+
+        public ApiInfoController(IConfigFileProvider configFileProvider)
+        {
+            _configFileProvider = configFileProvider;
+        }
+
+        [HttpGet("/api")]
+        [Produces("application/json")]
+        public object GetApiInfo()
+        {
+            return new
+            {
+                Latest = $"{_configFileProvider.UrlBase}/api/v3",
```

Comment:
`v3` is hardcoded here similar to initializeJSController.cs, IntegrationTestBase.cs and Startup.cs. Probably a good idea that we consolidate these into BuildInfo or somewhere at some point in a latest API prop.